en/docs/identity-providers/overview.md

```diff
@@ -0,0 +1,13 @@
+# External Identity Provider (IdP) Support for Choreo API Management
+
+Choreo provides admin permissions to privileged users to configure different authorization servers as Identity Providers (IdPs) from the Choreo Console. This brings the capability of supporting multiple IdPs for a given API. Hence, users in these external userbases can get authenticated with Choreo by exchanging the external IdP token.
+
+## Token exchange
+
+[Token Exchange Grant Type](https://datatracker.ietf.org/doc/html/rfc8693) is a protocol for an HTTP and JSON-based Security Token Service by defining how to request and obtain security tokens from OAuth 2.0 authorization servers. With this capability, you can exchange a JSON Web Token (JWT) issued by an external IdP for a Choreo token. (For more information, see [Generate Access Tokens using an External Identity Provider]())
```

Comment:
Seems like "[Token Exchange Grant Type]" should be changed to "[Token exchange specification]".  Please confirm.
Also, seems like the link is missing in the last sentence "see [Generate Access Tokens using an External Identity Provider]())"

Reply:
Thank you for pointing out the missing link. The link should be there to point to the page exchange-an-access-token-from-an-external-identity-provider.md. (The topic of that particular page should come here instead of "Generate Access Tokens using an External Identity Provider")

Reply:
@Nirothipan @vithu30 is it okay to change **Token Exchange Grant Type** to **Token exchange specification**?

Reply:
If we want to mention "[Token Exchange Grant Type]" how about the following
"Choreo supports the token exchange grant type, which uses the protocol defined in the [OAuth 2.0 token exchange specification](https://datatracker.ietf.org/doc/html/rfc8693)."
or something along those lines and then continuing with the description?

Reply:
" [Token exchange specification] defines a protocol for an ....."  shall we conclude like this